mass_mailing_switzerland/models/mass_mailing_contact.py

```diff
@@ -45,6 +242,11 @@ def get_partner(self, email):
     @api.multi
     def write(self, values):
         out = super().write(values)
+        if "partner_ids" in values:
+            for contact in self.filtered(lambda c: c.partner_ids):
+                contact.compute_salutation()
+                contact.compute_sponsored_child_fields()
+        self.action_update_to_mailchimp()
```

Comment:
```suggestion
            contact.with_delay().update_all_merge_fields_job(contact.partner_ids[0].id)
```
update should be handle trough cron